{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -36,6 +36,8 @@ ENV PATH=/root/.local/bin:$PATH
 
 COPY --from=base-image /root/ /root/
 
+RUN grep psycopg2 requirements.txt && apk add --no-cache libpq || true
```

Comment:
this should only happen on alpine, right?

Reply:
Oh, yeah. Didn't surface during testing as `|| true` did the trick, but a more elegant solution is to add `if`.